comp/api/api/apiimpl/observability/logging.go

```diff
@@ -34,28 +33,23 @@ func getLogFunc(code int) logFunc {
 
 // LogResponseHandler is a middleware that logs the response code and other various information about the request
 func LogResponseHandler(servername string) mux.MiddlewareFunc {
-	return logResponseHandler(servername, getLogFunc)
+	return logResponseHandler(servername, getLogFunc, clock.New())
 }
 
 // logResponseHandler takes getLogFunc as a parameter to allow for testing
-func logResponseHandler(serverName string, getLogFunc func(int) logFunc) mux.MiddlewareFunc {
+func logResponseHandler(serverName string, getLogFunc func(int) logFunc, clock clock.Clock) mux.MiddlewareFunc {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			lrw := negroni.NewResponseWriter(w)
-			start := time.Now()
-			next.ServeHTTP(lrw, r)
-			duration := time.Since(start)
+			var code int
```

Comment:
Could rename it to statusCode? I think it explain better what are we storing in it